features/NewCustomerRegistration.feature

```diff
@@ -0,0 +1,28 @@
+Feature: New customer Registration
+
+
+Background: customer Registration
+	Given the user navigates to Homepage
+    And clicks on login create a account button
+    
+Scenario: Empty Customer Registration Authentication (Negative scenario)
+	Given the user should be navigated to customer account page
+    When the user clicks on create an account button
+    Then the user should see 5 error message "This is a required field"
+
+Scenario: Emailid and Password validation for Customer Registration Authentication (Negative scenario)
```

Comment:
@karthikeyan7010 We can split this into multiple scenarios 1 for email & 1 for password. Each scenario should have multiple test inputs